sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -79,7 +79,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      * @param resumeToken A previously generated token used to resume the polling of the
      * operation.
      * @param client A #CertificateClient that is used for getting status updates.
-     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @param context A Azure::Core::Context controlling the request lifetime.
```

Comment:
Missing anchor tag `#`
```suggestion
     * @param context A #Azure::Core::Context controlling the request lifetime.
```

Reply:
Anchor tags can't span packages :(.